parquet-column/src/main/java/org/apache/parquet/column/page/PageReadStore.java

```diff
@@ -43,6 +43,14 @@
    */
   long getRowCount();
 
+  /**
+   *
+   * @return the row index offset of this row group.
+   */
+  default Optional<Long> getRowIndexOffset() {
```

Comment:
@shangxinli I added this new method in the interface. This leads to build issues from japicmp-maven-plugin - METHOD_ADDED_TO_INTERFACE.

I provided a default value for this method and this seems to solve the issue. Please provide your feedback on this approach/any alternate approach?